scalapbc/src/main/scala/com/trueaccord/scalapb/ScalaPBC.scala

```diff
@@ -1,17 +1,22 @@
 package com.trueaccord.scalapb
 
+import java.io.File
+
 import protocbridge.ProtocBridge
 
 import _root_.scalapb.ScalaPbCodeGenerator
 
 case class Config(
   version: String = "-v310",
   throwException: Boolean = false,
-  args: Seq[String] = Seq.empty)
+  args: Seq[String] = Seq.empty,
+  customProtocLocation: Option[String] = None)
 
 class ScalaPbcException(msg: String) extends RuntimeException(msg)
 
 object ScalaPBC {
+  private val customPathArgument =  "--protocPath="
```

Comment:
Let's make it `--protoc=` (without the `Path`)

Reply:
done